libs/rxidb/README.md

```diff
@@ -0,0 +1,220 @@
+const store$: Observable<IDBObjectStore> = database$.pipe(getObjectStore('example_store'));
+
+// set up the read stream.
+const exampleKey$: Observable<string> = store$.pipe(read('example_key'));
+
+exampleKey$.subscribe(console.log); // emits every time the value changes.
+
+// exampleKey$ will emit 4 times;
+store$
+  .pipe(
+    setItem('example_key', 'example_value_1'),
+    setItem('example_key', 'example_value_2'),
+    setItem('example_key', 'example_value_3'),
+    deleteItem('example_key')
+  )
+  .subscribe();
+```
+
+#### keys() & entries()
+
+You can get all the keys of a database using the `keys()` operator. This will return an `Observable` of all the keys in the store. With the `entries()` operator you can get all the values of the store.
+
+```typescript
+import { connectIndexedDb, getObjectStore, addItem, keys, entries } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one with autoIncrement.
+const store$: Observable<IDBObjectStore> = database$.pipe(
+  getObjectStore('example_store', { autoIncrement: true })
+);
+
+// returns an Observable of all the keys in the store.
+const keys$: Observable<IDBValidKey[]> = store$.pipe(keys());
+
+// returns an Observable of all the values in the store.
+const entries$: Observable<any[]> = store$.pipe(entries());
+
+keys$.subscribe(console.log); // emits every time the keys in the database change. (add a new key, remove a key, etc)
+entries$.subscribe(console.log); // emits every time the values change. Either by adding a new value, or by updating or deleting an existing value.
+```
+
+### Deleting databases
+
+The `deleteDatabase()` operator can be used to delete a database. This will complete the existing observables subscribed to database events.
+
+```typescript
+import { connectIndexedDb, getObjectStore, read, setItem, deleteItem } from '@this-dot/rxidb';
+import { Observable } from 'rxjs';
+
+// Connect to an IndexedDB database (create if the database does not exist)
+const database$: Observable<IDBDatabase> = connectIndexedDb('example_database');
+// Create an Object Store or get an existing one.
+const store$: Observable<IDBObjectStore> = database$.pipe(getObjectStore('example_store'));
+
+// set up the read stream.
+const exampleKey$: Observable<string> = store$.pipe(read('example_key'));
+
+exampleKey$.subscribe(console.log); // emits every time the value changes, and emits a `null` value when the database is deleted.
+
+// exampleKey$ will emit 4 times;
+store$
+  .pipe(
+    setItem('example_key', 'example_value_1'),
+    setItem('example_key', 'example_value_2'),
+    setItem('example_key', 'example_value_3'),
+    deleteItem('example_key')
+  )
+  .subscribe();
+
+// deletes the database
+deleteDatabase('example_database').subscribe();
```

Comment:
We should specify in the docs what happens if the database to delete doesn't exist.

Reply:
It did nothing, but I updated it to throw an error.